dockerize-app/dockerize-django
# Add initial files

#### What does this PR do?
This PR creates the initial Django app and makes the library `pip installable` by adding a `setup.py` as well. This addresses #1 .

#### How should this be manually tested?
Clone the repo (it's not on PyPy yet 😄) and `cd` into it. Make sure you create an environment and install Django 1.6 and above then run `pip install .`. That should complete successfully without errors or warnings. Running `pip freeze` should show `dockerize-django==0.1` in the list of dependencies.
